.github/workflows/drive.yml

```
@@ -42,12 +46,18 @@ jobs:
             fs.writeFileSync(`${process.env.GITHUB_WORKSPACE}/pr_id.zip`, Buffer.from(download.data));
 
       - name: 'Unzip artifact (PR ID)'
-        run: unzip pr_id.zip
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2219:style:2:1: Instead of 'let expr', prefer (( expr )) 

(shellcheck)